packages/vm/src/types.ts

```diff
@@ -137,11 +136,6 @@ export interface VMOpts {
    * Default: `false`
    */
   activatePrecompiles?: boolean
-  /**
-   * A genesisState to generate canonical genesis for the "in-house" created stateManager if external
-   * stateManager not provided for the VM, defaults to an empty state
-   */
-  genesisState?: GenesisState
```

Comment:
I have checked, we have 12-15 places where we do internally and only 1 where we use this `genesisState` functionality and otherwise we simply do:

```ts
this.vm.stateManager.generateCanonicalGenesis(genesisState)
```

So I thought this is as convenient (for a rarely used functionality) and from a `StateManagerInterface` perspective it had the advantage that we can lower the requirements for the `generateCanonicalGenesis()` implementations (since not necessary in VM) and do not need extra VM checks.

Reply:
Great, agreed